src/core/use_cases/get_all_items_use_case.py

```diff
@@ -0,0 +1,6 @@
+from src.infrastructure.persistance.item_datasource import ItemDatasource
+from typing import List
+from src.core.domain.entities.item import Item
+
+def get_all_items_use_case() -> List[Item]:
+    return ItemDatasource().get_all()
```

Comment:
Something to comment on, not to implement...

How would you sort or filter this request?